src/components/main/Logo.jsx

```diff
@@ -0,0 +1,15 @@
+import PropTypes from "prop-types";
+import {ReactComponent as DeleLogo} from './logo.svg';
+
+export function Logo({size}) {
+    return (
+        <DeleLogo height={
+            (size === 'sm') ? "32px" :
```

Comment:
Much better if you put this value into a variable, then pass as a prop

```
let size

switch (size) {
  case 'sm':
    ...
}
```

then `<DeleLogo {...{size}}/>`